libs/designsystem/src/lib/scss/_global-styles.scss

```diff
@@ -35,6 +35,10 @@
 
 $backdrop-opacity: 0.4;
 
+body.allow-modal-scroll {
```

Comment:
Consider renaming to `allow-background-scroll` or similar.

Reply:
(modal scroll sounds more like the content in the modal, to me at least)

Reply:
Ditto! :)